common/workunit/workunit.hpp

```diff
@@ -1855,4 +1855,14 @@ class WORKUNIT_API WuidPattern
     inline operator StringBuffer& () { return pattern; }
 };
 
+inline cost_type getGuillotineCost(IConstWorkUnit *wu)
+{
+    cost_type wuMaxCost = money2cost_type(wu->getDebugValueReal("maxCost", 0));
```

Comment:
picky/style: supplying default (0) here (we don't normally for these), but not supplying default on lines below.

Reply:
The ones below allow default values to missed out.  getDebugValueReal doesn't. @jakesmith

Reply:
ah ok.